service/src/msr_data_skx.json

```diff
@@ -762,7 +815,7 @@
                     "behavior":  "variable",
                     "aggregation": "average",
                     "writeable": true,
-                    "description": "Disable for the L2 hardware prefetcher."
+                    "description": "Disable for the L2 hardware prefetcher. When reading at a higher level domain than its native domain, it aggregates as the count of all such bits that have been set."
```

Comment:
Some of these don't have the explicit `expect_same` type. Not a functional problem, just checking whether that's intentional.

Reply:
Nope! This was set by mistake by my script. Thanks for catching this!

Reply:
Actually, why were these set as "average"? It seems like we can set them to "sum" (as we're doing with the other bool fields).

Reply:
Yeah, this should work like those other indicators using sum.